src/store/auth/actions.js

```diff
@@ -0,0 +1,32 @@
+import {
```

Comment:
about an actions and reducers. In current project I try to use `https://redux-toolkit.js.org/`. This lib has helpers `createActions` and `createReducer`. Try to use in this way